pkg/util/interpreter/rule.go

```diff
@@ -367,14 +367,14 @@ func (d *dependencyInterpretationRule) Document() string {
 The script should implement a function as follows:
 function GetDependencies(desiredObj)
   dependencies = {}
-  if desiredObj.spec.serviceAccountName ~= nil and desiredObj.spec.serviceAccountName ~= "default" then
```

Comment:
I believe this is not a bug. It's more like an update to the documentation.

Reply:
+1

There are other places throughout the codebase that need to be changed as well.